ci/run.yml

```diff
@@ -0,0 +1,38 @@
+- hosts: controller
+  vars:
+    fvt_dir: "{{ ansible_env.HOME}}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}"
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
```

Comment:
```suggestion
    PATH: "{{ cifmw_path if cifmw_path is defined else ['~/.crc/bin:~/.crc/bin/oc:~/bin', ansible_env.PATH ] | join(':') }}"
```

Reply:
I highly recommend using this workaround for now, and figuring out later where the cifmw_path is set, once the rest is working.